### ISSUE TITLE
-Wsingle-bit-bitfield-constant-conversion in fs/f2fs

### ISSUE THREAD

**Issue by nathanchance** [ClangBuiltLinux/linux#1796]
```
../fs/f2fs/data.c:995:17: error: implicit truncation from 'int' to a one-bit wide bit-field changes value from 1 to -1 [-Werror,-Wsingle-bit-bitfield-constant-conversion]
        fio->submitted = 1;
                       ^ ~
../fs/f2fs/data.c:1011:15: error: implicit truncation from 'int' to a one-bit wide bit-field changes value from 1 to -1 [-Werror,-Wsingle-bit-bitfield-constant-conversion]
                        fio->retry = 1;
                                   ^ ~
2 errors generated.

../fs/f2fs/segment.c:3320:16: error: implicit truncation from 'int' to a one-bit wide bit-field changes value from 1 to -1 [-Werror,-Wsingle-bit-bitfield-constant-conversion]
                fio->in_list = 1;
                             ^ ~
1 error generated.
```

Patch submitted: https://lore.kernel.org/20230201-f2fs-fix-single-length-bitfields-v1-1-e386f7916b94@kernel.org/

**Comment by nathanchance** [comment]
Now fixed in the maintainer's tree: https://git.kernel.org/jaegeuk/f2fs/c/2eae077e6e46f9046d383631145750e043820dce

**Comment by nathanchance** [comment]
Now fixed in next-20230207.
